samples/net/cloud/aws_iot_mqtt/boards/ek_ra8m1.conf

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) 2024 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_ENTROPY_DEVICE_RANDOM_GENERATOR=y
```

Comment:
this is on by default, if the device supports it. so no need to enable it explicit.

Reply:
Thank you, I rebase the change and remove the commit for this.